ci_framework/hooks/playbooks/validate_podified_deployment.yml

```diff
@@ -0,0 +1,24 @@
+---
+- hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Make sure Nova api service is up and running
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_login_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
+        oc get pods -n openstack --selector service=nova-api -o jsonpath={.items[*].status.phase}
```

Comment:
is that namespace always the same? iirc there's a NAMESPACE parameter one may set in install_yamls (and available in cifmw_install_yamls_defaults dict I think).